lib/clearance/configuration.rb

```diff
@@ -133,6 +133,16 @@ def user_id_parameter
     def routes_enabled?
       @routes
     end
+
+    # Reloads the clearance user model class.
+    # This is called from the Clearance engine to reload the configured
+    # user class during each request while in development mode.
```

Comment:
The comment in the commit msg about this happening only on startup in non-development environments was interesting / informative. Should we include that info here as well?